cli/examples/custom-commit-templater/main.rs

```diff
@@ -78,6 +126,10 @@ impl CommitTemplateLanguageExtension for HexCounter {
 
         table
     }
+
+    fn build_cache_extensions(&self, extensions: &mut AnyMap) {
+        extensions.insert(MostDigitsInId::new());
```

Comment:
Perhaps, `.insert()` should panic if the type already exists. That's the point of adding separate initialization phase.

Reply:
Done.